README.md

```diff
@@ -163,7 +163,7 @@ type: markdown
 content: >-
   <h2 class="some"> <ha-icon icon='mdi:shield-half-full'></ha-icon> Watchman report</h2>
   <h3> Missing Services: {{ states.sensor.watchman_missing_services.state }} </h3>
-  {%- for item in state_attr("sensor.watchman_missing_services", "services") %}
+  {%- for item in state_attr("sensor.watchman_missing_services", "services") or () %}
```

Comment:
```suggestion
  {%- for item in state_attr("sensor.watchman_missing_services", "entities") or () %}
```
resolves merge conflict

Reply:
Actually this one should say services as the first example block is for entities. Not sure if you can edit again?

Reply:
Unfortunately watchman stores the missing services in attribute `entities` (see screenshot below). I agree that this should be `services` but it in fact it is not and changing watchman to store it in a `services` attribute would cause a breaking change - something I'd like to avoid for now.

![image](https://github.com/andreasbrett/thewatchman/assets/6610451/794d1d4b-20aa-4464-aedc-0e6674ba30e6)